testdata/project-v2/go.mod

```diff
@@ -1,12 +1,11 @@
 module sigs.k8s.io/kubebuilder/testdata/project-v2
 
-go 1.12
+go 1.13
```

Comment:
Please revert this change.

Reply:
^ the same

Reply:
you need to update the scaffolding file too

Reply:
Done @DirectXMan12 :-) 
Thank you for the good spot 🥇